### ISSUE TITLE
[BUG] Migration 3.0 to 3.1 $socket_in(af) empty

### ISSUE THREAD

**Issue by hamid-elaosta** [OpenSIPS/opensips#2185]
**OpenSIPS version you are running**
```
$ opensips -V
version: opensips 3.1.0 (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, HP_MALLOC, DBG_MALLOC, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
main.c compiled on  with gcc 7
```

**Describe the bug**
Attempting to migrate  working system from 3.0.1 to 3.1.0 by making script updates according to the documentation at: https://www.opensips.org/Documentation/Migration-3-0-0-to-3-1-0

Opensips starts and shows no syntax errors but, when attempting to obtain `$socket_in(af)` the value is empty, and a pv error is shown (not sure if it's related):
```
Jul 28 09:38:34 [24] ERROR:core:pv_value_print: unknown type 0
Jul 28 09:38:34 [24] 1 socket_in: 
Jul 28 09:38:34 [24] ERROR:core:pv_value_print: unknown type 0
```
The xlog line which printed the statement above:
```
xlog("1 socket_in: $socket_in(af)\n");
```

**To Reproduce**
1. Start with working OpenSIPs 3.0.x configuration using `$af`
2. Update OpenSIPs to 3.1.0-1 using the apt repository for bionic
3. Replace `$af` with `$socket_in(af)` and start server and `listen=` with `socket=` and any other change required from the document.
4. Attempt to log/access the value of `$socket_in(af)`, which will be empty/blank.
5. Attempting to read `$socket_in(af)` from multiple route blocks, it is always empty.

**Expected behavior**
Value of `$socket_in(af)` should be "INET" for IPv4 or "INET6" for IPv6 sockets.

**Relevant System Logs**
N/A

**OS/environment information**
 - Operating System: Ubuntu 18.04 in Docker Container
 - OpenSIPS installation: apt.opensips.org bionic

**Additional context**
 - **listen socket is TLS**
- EDIT: Also tested with TCP listen socket, with the same result.

**Comment by bogdan-iancu** [comment]
Thank you @hamid-elaosta  for the report, the issue was fixed, please see the GIT repo or the nightly generated packages.

**Comment by hamid-elaosta** [comment]
@bogdan-iancu Thank you, amazing as always! I'll test out the nightly to confirm it's all fixed.

**Comment by hamid-elaosta** [comment]
Confirmed fixed in `3.1.0~20200728~65c54c90b-1`.
